pkg/config/controller_config.go

```diff
@@ -77,6 +82,12 @@ func configInit(metadata EC2Metadata) error {
 	// CLUSTER_LOCAL_GATEWAY
 	DefaultServiceNetwork = os.Getenv(CLUSTER_LOCAL_GATEWAY)
 
+	// CLUSTER_NAME
+	ClusterName, err = getClusterName(sess)
+	if err != nil {
+		return fmt.Errorf("cannot get cluster name: %s", err)
+	}
+
```

Comment:
Does this `configInit` also work for ECS Fargate based  k8s cluster?

Reply:
env or metadata same as other variables